docker-compose.yml

```diff
@@ -2,13 +2,15 @@ version: "2"
 
 services:
   pg:
-    image: postgres:9.3
+    image: postgres:10.14
```

Comment:
ah, good catch! should we go all the way to 10.15 since that's what we're using elsewhere?

Reply:
I'm seeing Postgres version `10.14` still being used in Conjur, e.g.:
https://github.com/cyberark/conjur/blob/master/dev/docker-compose.yml#L4
Should this be `10.15` everywhere?

Reply:
I changed this to version `10.15` to match what's used in cyberark/conjur-oss-helm-chart.